doc/lettable-operators.md

```diff
@@ -4,6 +4,8 @@ Starting in version 5.5 we have shipped "lettable operators", which can be acces
 
 **NOTE**: During 5.5 beta we will be bikeshedding a few of the names for operators that we had to give new names to due to keyword restrictions in javascript. 
 
+**NOTE**: If you're using TypeScript, version >= 2.4 is required, otherwise lettable operators will lose the type and compilation will break. 
+
```

Comment:
This is a little more "doom and gloom" than I'd like.. It sounds like you can't use TypeScript 2.3 or under, which isn't true. It should just be `NOTE: in TypeScript 2.3 and under, typings will need to be added to functions passed to operators` or something like that.

Reply:
Even if it's true technically, in real world no one will do this. You really need to add the type to each functions, sometimes even twice, so that's clearly not usable. And in some way it's really an error, as lettable operators transfer the wrong type in TS < 2.4, then causing real compilation errors when you do any action on the data. But you're the owner, you choose what's best.